src/sampling/fibonacci.jl

```diff
@@ -0,0 +1,123 @@
+"""
+	FibonacciSampling(n)
+
+Generate samples on a sphere using the Fibonacci lattice method.
+The parameter `n` specifies the number of points to generate.
+
+## Example
+
+Sample a sphere with 1000 points:
+
+```julia
+sample(Sphere((0,0,0), 1), FibonacciSampling(100))
+"""
+struct FibonacciSampling <: SamplingMethod
+  size::Int
+  function FibonacciSampling(size::Int)
+    if size ≤ 0
+      throw(ArgumentError("Size must be positive"))
+    end
+    new(size)
+  end
+end
+
+function sample(::AbstractRNG, geom::Union{Sphere,Ball{𝔼{3}}}, method::FibonacciSampling)
+
+  # Compute x,y samples from regular sphere of radius 1
```

Comment:
```suggestion
  # compute x,y samples from regular sphere of radius 1
```
comments in lowercase